From 31ea9c508ba3f70a42ad57f854613cc2d57bf809 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:03:58 -0400
Subject: [powerpc] iommu: Remove unused pci_iommu_init() and pci_direct_iommu_init()

Message-id: <1410545655-205645-10-git-send-email-dzickus@redhat.com>
Patchwork-id: 94097
O-Subject: [RHEL7 PATCH 009/626] powerpc/iommu: Remove unused pci_iommu_init() and pci_direct_iommu_init()
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 5524f3fc069b6435f9ff0db573e3a8b5082ef528
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Tue Jun 11 13:57:05 2013 -0600

    powerpc/iommu: Remove unused pci_iommu_init() and pci_direct_iommu_init()

    pci_iommu_init() and pci_direct_iommu_init() are not referenced anywhere,
    so remove them.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/asm/iommu.h b/arch/powerpc/include/asm/iommu.h
index cbfe678..e670f73 100644
--- a/arch/powerpc/include/asm/iommu.h
+++ b/arch/powerpc/include/asm/iommu.h
@@ -125,13 +125,6 @@ extern void iommu_init_early_pSeries(void);
 extern void iommu_init_early_dart(void);
 extern void iommu_init_early_pasemi(void);
 
-#ifdef CONFIG_PCI
-extern void pci_iommu_init(void);
-extern void pci_direct_iommu_init(void);
-#else
-static inline void pci_iommu_init(void) { }
-#endif
-
 extern void alloc_dart_table(void);
 #if defined(CONFIG_PPC64) && defined(CONFIG_PM)
 static inline void iommu_save(void)
-- 
1.7.1